life.bolls.bolls.yaml

```diff
@@ -41,4 +41,6 @@ modules:
         path: life.bolls.bolls.appdata.xml
       - type: file
         filename: linux-unpacked.tar.xz
-        path: linux-unpacked.tar.xz
+        url: https://storage.googleapis.com/resurrecting-cat.appspot.com/linux-unpacked.tar.xz
```

Comment:
this file should be versioned otherwise if it changes the hash won't match any-more failing the build.

Reply:
Yeah, I know. I will version it from the next release.